nengo_spa/network.py

```diff
@@ -166,9 +166,6 @@ def get_network_input(self, name):
         except KeyError:
             raise SpaNetworkError("Could not find network input %r." % name)
 
-    def get_input_vocab(self, name):
```

Comment:
Just to make sure I'm understanding this change, only the associative memory has an input vocab, so now all the networks have vocab attributes instead? Consequently, these accessors aren't required?